test/e2e/pvc_label_selector.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
yes this is just a function test for selector-label volume binding feature. not vmware specific. but as test is using vsphere cloud provider to create vmdk, we are skipping execution for this test when provider is not vsphere.